svthalia/concrexit
# Non nix env files

### Summary
Automatically place env files from the repo on the server on staging deploy

### How to test
Steps to test the changes you made:
1. Merge this
2. Check the github actions output
3. The rsync task should have run succesfully
